src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -81,6 +91,7 @@ public synchronized static void initializeOpenShiftClient(String serverUrl) {
   }
 
   public synchronized static OpenShiftClient getOpenShiftClient() {
+    setOpenShiftOauthToken();
```

Comment:
ahh ... yep, missed that, and went to look at the current source  ... agreed, seems like infinite loop is possible